pkg/dumper/query/dumper.go

```diff
@@ -50,37 +49,40 @@ func (d *textDumper) Dump() error {
 
 		// Create read/write chanel
 		rowChan := make(chan *database.Row)
-
 		go d.reader.ReadTable(tbl, rowChan)
 
-		for {
-			rowFromChan := <-rowChan
-			if rowFromChan == nil {
-				break
-			}
-			row := *rowFromChan
-
-			io.WriteString(d.output, insert)
-			io.WriteString(d.output, "(")
-			for i, column := range columns {
-				data := row[column]
-
-				if i > 0 {
-					io.WriteString(d.output, ",")
+		go func() {
+			for {
+				rowFromChan, more := <-rowChan
```

Comment:
Nice did not know about this (aka related to https://tour.golang.org/concurrency/4)